app/services/crash-reporter.ts

```diff
@@ -27,6 +27,7 @@ enum EAppState {
 
 interface ICrashReporterState {
   code: EAppState;
+  detected: string; // what was detected before close happens: ipc freez, backend crash, electron window unresponcive
```

Comment:
I would rephrase the comment to: `The Backend event that was detected when the app close event was triggered: IPC freeze or backend crash or electron window unresponsive.`